WolvenKit.App/ViewModels/Documents/IDocumentViewModel.cs

```diff
@@ -4,6 +4,20 @@
 
 namespace WolvenKit.App.ViewModels.Documents;
 
+public class SaveAsParameters
```

Comment:
Guess this is based on my previous feedback?
Maybe should either remove it for now or implement it completly?
Currently it and (`IDocumentViewModel.SaveAs()`) is not used anywhere.

Reply:
It is, yes. I think it became unnecessary because I'm writing the cr2w directly now, so I'll remove it!